var/spack/repos/builtin/packages/recon/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Recon(MakefilePackage):
+    """RECON: a package for automated de novo identification of repeat families
+       from genomic sequences
+
+       NOTE: The 1.08 is a patched version from the RepeatModeler developers.
+             The original versions can be found on the homepage."""
+
+    homepage = "http://eddylab.org/software/recon/"
+    url      = "http://www.repeatmasker.org/RepeatModeler/RECON-1.08.tar.gz"
+
+    version('1.08', sha256='699765fa49d18dbfac9f7a82ecd054464b468cb7521abe9c2bd8caccf08ee7d8')
+
+    build_directory = 'src'
+    build_targets = ['install']
```

Comment:
The way the `MakefilePackage` base class works is that it first runs `make <build_targets>` then `make <install_targets>`. Currently, the default build targets are empty and the default install targets are `install`. If you want to add a symlinking step after installation, you can do something like:
```python
@run_after('install')
def symlink_foo(self):
    print(self.prefix)
    # do stuff
```

Reply:
I understand -- recon has a very basic makefile which does very little "installing" from the install target and this was a bit of a workaround. I refactored the recipe more to use the proper decorators.